copier/main.py

```diff
@@ -187,6 +188,7 @@ class Worker:
     unsafe: bool = False
 
     answers: AnswersMap = field(default_factory=AnswersMap, init=False)
+    _cleanup_hooks: List[Callable] = field(default_factory=list, init=False)
```

Comment:
I _think_ you can wrap the `Callable` type hint with Pydantic's `SkipValidation` container (https://docs.pydantic.dev/latest/usage/validators/#special-types):

```suggestion
    _cleanup_hooks: List[SkipValidation[Callable]] = field(default_factory=list, init=False)
```

Don't forget to import it from the `pydantic` package.

I haven't tried it though.

Reply:
Oh, wait a second. It's about serialization, not validation. 🤔

Reply:
I think this was wrong, so I'm resolving this conversation as invalid.